doc/nvim-tree-lua.txt

```diff
@@ -859,6 +904,11 @@ UI rendering setup
             Glyph for symlinks to files.
               Type: `string`, Default: `""`
 
+            *nvim-tree.renderer.icons.glyphs.modified*
+            Icon to display for modified files. Can be multiple characters if
+            |renderer.icons.modified_placement| isn't `"signcolumn"`.
```

Comment:
That's quite nice. We will need to do some extensive testing of `get_padding` as that is quite... fragile.

We could allow multiple only for `"after"` to avoid that. It would be consistent with vim's `[+]` after which I imagine is the use case.